packages/commonwealth/server/controllers/server_threads_methods/create_thread.ts

```diff
@@ -55,43 +55,34 @@ export type CreateThreadResult = [
   TrackOptions
 ];
 
-export async function __createThread({
-  user,
-  address,
-  chain,
-  title,
-  body,
-  kind,
-  readOnly,
-  topicId,
-  topicName,
-  stage,
-  url,
-  attachments,
-  canvasAction,
-  canvasSession,
-  canvasHash,
-  discord_meta,
-}: CreateThreadOptions): Promise<CreateThreadResult> {
+export async function __createThread(
+  this: ServerThreadsController,
```

Comment:
Any reason we're now passing a `this` var?

Reply:
It's for the TS intellisense in code editors so they know what the `this` value is for the function.

I forgot to add it previously so went ahead and did it in this PR.

Reply:
It doesn't functionally change anything btw, it's strictly for typing purposes